packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -268,6 +270,28 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     }
   }
 
+  public getStartItemIndexInView(): number {
+    const pages = this.state.pages || [];
+    for (const page of pages) {
+      const isPageVisible =
+        page.key &&
+        !page.key.startsWith(SPACER_KEY_PREFIX) &&
+        (this._scrollTop || 0) >= page.top &&
+        (this._scrollTop || 0) <= page.top + page.height;
+      if (isPageVisible) {
+        const startIndexStr = page.key.replace(PAGE_KEY_PREFIX, '');
```

Comment:
an `IPage` has a `startIndex` property, why are you doing string magic here?

Reply:
fixed